stdlib/public/stubs/LibcShims.cpp

```diff
@@ -23,7 +23,9 @@
 
 #include <stdio.h>
 #include <sys/types.h>
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
```

Comment:
Alternative:

```cpp
#if __has_include(<unistd.h>)
#include <unistd.h>
#endif
```

Reply:
@jrose-apple - `__APPLE__` doesn't really identify macOS X unfortunately.  Technically, it could be macOS classic, so it is for portability.  That said, I really do wonder why `__unix__` is not just defined for macOS X.